### ISSUE TITLE
HM: locus equation

### ISSUE THREAD

**Issue by urapadmin** [arch-kiosk/arch-kiosk-office#2526]

**Comment by urapadmin** [comment]
see also #1790

**Comment by lbestock** [comment]
A locus equation of the once had been one (Harris himself has that as a thing - he calls it "once whole") would ideally only get one node, no? And the node would be called something like FA-003=FA-006. Or two nodes next to each other but connected by an = sign (I have seen this in practice with some actual use case HMs).

Harris himself does not allow for any contemporary relations at all except equivalences. That does help simplify and makes clearer that nodes on a horizontal line are not contemporary. But I do not like not having bonding walls as contemporary, and I would not want them to be one locus or an equivalence. 

I am trying to think through the implications of this (again, now with HMs on the mind). Chronologically equivalent loci that are still distinct loci could have different locus relations than one another, but only on a physical level, not on a chronological/HM level. Loci that once were whole I started thinking could not have separate physical relations but that's not in fact true. A deposit cut into two by the cutting of a foundation trench could for instance have different and importantly so relations with what came after the building of the wall. But those differences still wouldn't be chronological.

**Comment by urapadmin** [comment]
no, I think we would rather go for equivalences and otherwise temporally interlocked loci. As for equivalences: If we mush the nodes together, they must still be independent objects to click on because we want our HM tool one day to manipulate their physical relations and those stay separate (unless we go back to the merging discussion, but let's not in this context). One also must not forget that they can be many and not only two that are together. This is difficult for the UI. I would have to treat them as one node but a node can only be as wide as all other nodes. So I have to visualize that this node is in fact a conglomerate of nodes outside of the tool.

Other contemporary relations would just stay as they are and all the relation does is it makes sure that these nodes happen to be in the same row.

**Comment by urapadmin** [comment]
- for reference: see #1790
